README.md

```diff
@@ -49,9 +49,11 @@ You can dive into `build-ventus.sh` file to see the detailed information about b
 
 ### 3: Bridge icd loader
 
-Run `export LD_LIBRARY_PATH=<path_to>/ventus-llvm/install/lib` to tell OpenCL application to use your own built `libOpenCL.so`, also to correctly locate LLVM shared libraries
+Run `export VENTUS_INSTALL_PREFIX=<path_to>/ventus-llvm/install` to set `VENTUS_INSTALL_PREFIX` environment variable(system environment variable recommended)
```

Comment:
Users can use the tool chain without the "ventus-llvm" directory, such as the release version. Is it more appropriate to remove "ventus-llvm"?